src/encord_active/cli/main.py

```diff
@@ -73,23 +75,105 @@ def download(
+        help="The root directory of the dataset you are trying to import",
+        file_okay=False,
+    ),
+    glob: List[str] = typer.Option(
+        ["**/*.jpg", "**/*.png", "**/*.tiff"],
+        "--glob",
+        "-g",
+        help='Glob pattern to choose files in the "leaf directories". Note that you can repeat the `--glob` argument if you want to match multiple things.',
+    ),
+    target: Path = typer.Option(
+        Path.cwd(),
+        "--target",
+        "-t",
+        help="Directory where the project would be saved.",
+        file_okay=False,
+    ),
+    project_name: str = typer.Option(
+        "",
+        "--name",
+        "-n",
+        help="Name to give the new project. If no name is provided, the root directory will be used with '[EA] ' prepended",
+    ),
+    symlinks: bool = typer.Option(
+        False,
+        help="Use symlinks instead of copying images to the target directory.",
+    ),
+    dryrun: bool = typer.Option(
+        False,
+        help="Print the files that will be imported WITHOUT importing them.",
+    ),
+):
+    """
+    [bold]Initialise[/bold] a project from your local file system by searching for images based on the `glob` arguments.
+    By default, all jpeg, jpg, png, and tiff files will be matched.
+
+
+    """
+    from encord_active.lib.project.local import init_local_project
 
-[cyan]cd "{escape(cd_dir.as_posix())}"
-encord-active visualise
-    """,
-            title="🌟 Success 🌟",
-            style="green",
-            expand=False,
+    try:
+        project_path = init_local_project(
+            root=root, target=target, glob=glob, project_name=project_name, symlinks=symlinks, dryrun=dryrun
         )
-    )
+        if dryrun:
+            if not isinstance(project_path, list):
+                raise ValueError("Expected a list of paths for the dryrun execution")
+            file_paths = project_path
+
+            directories: Set[Path] = set()
+            for file in file_paths:
+                directories.add(file.parent)
+                print(file)
+            print()
+            rich.print(
+                Panel(
+                    f"Found {len(file_paths)} files in {len(directories)} directories.",
+                    title=":bar_chart: Stats :bar_chart:",
+                    expand=False,
+                )
+            )
+        else:
+            if not isinstance(project_path, Path):
+                raise ValueError("Expected a single path for an actual init execution")
+
+            success_with_visualise_command(project_path, "Project initialised :+1:")
+
+    except NoFilesFoundError as e:
+        rich.print(
+            Panel(
+                e.args[0],
+                title=":fire: No Files Found :fire:",
+                expand=False,
+                style="yellow",
+            )
+        )
+        typer.Abort()
+    except ProjectExistsError as e:
+        rich.print(
+            Panel(
+                f"""
+{e.args[0]}
```

Comment:
[nitpick] 💅

I think it's cleaner 

```suggestion
{str(e)}
```